src/plone/restapi/services/discussion/utils.py

```diff
@@ -28,13 +28,27 @@ def permission_exists(permission_id):
     return permission is not None
 
 
+def can_view(context):
+    """Returns true if current user has the 'View comments' permission."""
+    if not permission_exists("plone.app.discussion.ViewComments"):
```

Comment:
`View comments` permission is not yet in `plone.app.discussion`. It will be added after this pr is merged https://github.com/plone/plone.app.discussion/pull/191.
The default behaviour of comments viewlet from `plone.app.discussion` is that everyone that has `View` permission can see the comments.
So I added this fallback in case someone uses a version of `plone.app.discussion` in which `View comments` permission doesn't exist. Without the fallback `can_view` would be `False` if there is no `View comments` permission.